modules/40_techpol/NDC/datainput.gms

```diff
@@ -53,10 +53,10 @@ p40_FE_RenShare(tall,regi) = f40_FE_RenShare(tall,regi);  !! rescale unit from [
 p40_noncombust_acc_eff(t,"CHN",te)$(sameas(te,"spv") OR sameas(te,"csp") OR sameas(te,"wind") OR sameas(te,"windoff") OR sameas(te,"tnrs") OR sameas(te,"spv") OR sameas(te,"geohdr") OR sameas(te,"hydro")) = 0.38; !! substitution accounting for low-carbon electricity generation at coal efficiency of 38%
 p40_noncombust_acc_eff(t,"CHA",te)$(sameas(te,"spv") OR sameas(te,"csp") OR sameas(te,"wind") OR sameas(te,"windoff") OR sameas(te,"tnrs") OR sameas(te,"spv") OR sameas(te,"geohdr") OR sameas(te,"hydro")) = 0.38; !! substitution accounting for low-carbon electricity generation at coal efficiency of 38%
 *** lower bound on gas share in PE
-p40_PEgasBound("2020","CHN") = 0.1; 
-p40_PEgasBound(t,"CHN")$(t.val gt 2020) = min(0.1 + (t.val -2020) * 0.005	,0.2 - (t.val - 2040) * 0.005 ); !! rising to 20% in 2040 and then declining again, to allow for high LC shares (no bound on gas after 2080)
-p40_PEgasBound("2020","CHA") = 0.1; 
-p40_PEgasBound(t,"CHA")$(t.val gt 2020) = min(0.1 + (t.val -2020) * 0.005	,0.2 - (t.val - 2040) * 0.005 ); !! rising to 20% in 2040 and then declining again, to allow for high LC shares (no bound on gas after 2080)
+p40_PEgasBound("2020","CHN") = 0.08; 
+p40_PEgasBound(t,"CHN")$(t.val gt 2020) = min(0.08 + (t.val -2020) * 0.004	,0.1 - (t.val - 2030) * 0.004 ); !! rising to 10% in 2025 and 2030 and then declining to zero around 2050
+p40_PEgasBound("2020","CHA") = 0.08; 
```

Comment:
declines to zero in 2055, right?

Reply:
That was a mistake, corrected with the extra commit just now... (and thanks for catching this)